core/libLocator/vmLocator_windows.go

```diff
@@ -0,0 +1,17 @@
+// +build windows
+
+package vm
+
+import (
+	"os/user"
+)
+
+const libName = "libhera.dll"
+
+func WASMLibLocation() string {
+	usr, err := user.Current()
+	if err != nil {
+		return ""
+	}
+	return usr.HomeDir + "/elrond-vm-binaries/" + libName
```

Comment:
Should replace slashes with backslashes. Currently, the resulting path looks like this on Windows: `C:\Users\User/elrond-vm-binaries/libhera.dll`

Not critical, but not recommended.

Reply:
done.